### ISSUE TITLE
Staging Review finding: Masked SSN may be announced by screen readers in a confusing way

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104824]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that meets best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/ask-va/your-personal-information](https://staging.va.gov/contact-us/ask-va/your-personal-information) when authenticated, the user's Social Security number is displayed with the first five digits masked. Screen readers are inconsistent in how they handle privacy masking like this, but you may end up with screen readers either:
- Just announcing the last four digits ("Social Security number: 1234")
- Announcing the masked characters then the last four digits ("Social Security number: star-star-star-star-star 1234")

Neither of these is exactly a faithful representation of what's intended here.

<details><summary>Link, screenshot or steps to recreate</summary>

![Prefilled veteran information, including Social Security number. The first five digits have been masked using a black dot character.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/62304-Brian-5.png?raw=true)

</details>

## Recommended action

Something like this should give a more consistent experience, and also better match the sighted user experience:

```
<p class="vads-u-margin--1px dd-privacy-mask" data-dd-action-name="Veteran's SSN">Social Security number: <span aria-hidden="true">●●●–●●–1234</span> <span class="sr-only">ending in 1234</span></p>
```

This would be announced as "Social Security number: ending in 1234."

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
